eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1419,6 +1403,9 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
     // -> in CLOSING we either have mutual closed (so no more htlcs), or already have unilaterally closed (so no action required), and we can't be in OFFLINE state anyway
     case Event(c: CurrentBlockCount, d: HasCommitments) => handleNewBlock(c, d)
 
+    case Event(c: CurrentFeerates, d: HasCommitments) if nodeParams.onChainFeeConf.closeOnOfflineMismatch =>
```

Comment:
Alternatively we can do the check for `nodeParams.onChainFeeConf.closeOnOfflineMismatch` inside `handleOfflineFeerate` with the advantage that we can print a log line if the mismatch is over the threshold but the flag is disabled, WDTY @pm47 ?

Reply:
I think that's better, we want to see the warning even if we do nothing.